contracts/infrastructure/dapp/YearnV2Filter.sol

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2021  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.3;
+
+import "./BaseFilter.sol";
+
+/**
+ * @title YearnV2Filter
+ * @notice Filter used for deposits & withdrawals into/from YearnV2
+ * @author Olivier VDB - <olivier@argent.xyz>
+ */
+contract YearnV2Filter is BaseFilter {
+
+    // Note: deposit and withdraw method implementations in Vyper have default values for all their arguments
+    // (see https://github.com/yearn/yearn-vaults/blob/master/contracts/Vault.vy).
+    // Each Vyper method therefore results in N+1 ABI methods for N parameters
+    // These ABI methods are denoted (DEPOSIT|WITHDRAW){i} below where i is the number of parameters for the method
+
+    bytes4 private constant DEPOSIT0 = bytes4(keccak256("deposit()"));
+    bytes4 private constant DEPOSIT1 = bytes4(keccak256("deposit(uint256)"));
+   
+    bytes4 private constant WITHDRAW0 = bytes4(keccak256("withdraw()"));
+    bytes4 private constant WITHDRAW1 = bytes4(keccak256("withdraw(uint256)"));
+    bytes4 private constant WITHDRAW3 = bytes4(keccak256("withdraw(uint256,address,uint256)"));
```

Comment:
Shouldn't the filter accept `WITHDRAW2 = bytes4(keccak256("withdraw(uint256,address)"));` with the default `maxLoss` and block `WITHDRAW3` where one can pass a large `maxLoss` and potentially decrease the wallet's value?

Reply:
Blocking `WITHDRAW3` is a good idea, and means that whales may have to use guardians to liquidate large positions, which is reasonable.

`WITHDRAW1` already provides a default `maxLoss` so no need for `WITHDRAW2`.